docs/tutorials/seurat-de/seurat-de.md

```diff
@@ -951,7 +940,6 @@ FeaturePlot(ifnb.filtered, reduction = 'umap.cca',
 # Find upregulated genes in each group (cell type and condition)
 ifnb.treatVsCtrl.markers <- FindAllMarkers(ifnb.filtered,
                                           only.pos = TRUE)
-saveRDS(ifnb.treatVsCtrl.markers, "ifnb_stimVsCtrl_markers.rds")
```

Comment:
I don't think this line should be removed. If learners are running this bit of code, it's helpful for them to save the data artifact or they would need to run it again later.